src/main/scala/fi/aalto/cs/apluscourses/intellij/actions/ReplAction.scala

```diff
@@ -108,6 +110,18 @@ class ReplAction extends RunConsoleAction {
 
   }
 
+  def getDefaultModule(@NotNull project: Project): Option[Module] =
+    Option(ModuleManager.getInstance(project).findModuleByName(PluginSettings
+      .getInstance()
```

Comment:
how would you feel if there would be method wrappers like `PluginSettings.getInstance().getCourse().getAutoInstallComponentNames()` (and other shorthand methods alike)? :smiley:

Reply:
For unit-testability (and more loosely-coupled design in general), objects shouldn't access `PluginSettings` like singleton object instances but they should be given an interface via which they can access the required information in them.

`PluginSettings::getCourse` would need `Project` for its parameter and I'm not sure if it would be that useful as such.